streams.json

```diff
@@ -5313,22 +7062,52 @@
           "repository_type": "GIT",
           "repository_url": "https://git.eclipse.org/r/jdt/eclipse.jdt.core",
           "codebase": "master",
-          "tag": "4.6.1.redhat-1",
+          "tag": "R4_6_1",
+          "group_id": "org.eclipse.jdt.core.compiler",
+          "artifact_id": "ecj",
           "version": "4.6.1.redhat-1",
-          "gav": "org.eclipse.jdt.core.compiler",
+          "gav": "ecj:org.eclipse.jdt.core.compiler:4.6.1.redhat-1",
           "comment": "master is at 4.7, "
         },
+        {
+          "component_name": "eclipse-link",
```

Comment:
This has been an oversight, it is not in prod kb, but we can link it using the dictionary with this GA org.eclipse.persistence:eclipselink